### PR TITLE
compression tutorial

### PR DESCRIPTION
**Pull request by XiaoheYou**
add compression model tutorial

### CODE REVIEW

**Comment by XiaoheYou** [comment]
我已经收到邮件了，谢谢~

File: docs/source/tutorials/compression.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+- `CONFIG_PATH`: the config file path of a detection method
+
+- `WORK_DIR`: your path to save models and logs
+
+- `MODEL_PATH`: the quantized models
+
+- `PRUNING_CLASS`: pruning class for pruning models
+
+- `PRUNING_ALGORITHM`: pruning algorithm using by pruning class
```
Comment (wenmengzhou) [inline docs/source/tutorials/compression.md:84]: same as before, what are the valid values for these params

File: docs/source/tutorials/compression.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+python tools/quantize.py \
+		configs/detection/yolox/yolox_s_8xb16_300e_coco.py \
+		models/yolox_s.pth \
+		--pruning_class AGP \
+		--pruning_algorithm taylorfo
+```
+
+### Evaluation
+
+**Single gpu:**
```
Comment (wenmengzhou) [inline docs/source/tutorials/compression.md:101]: dose the output model support gpu?   quantized model can only support cpu

File: docs/source/tutorials/compression.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+Edit `data_root`path in the `${CONFIG_PATH}` to your own data path.
+
+```shell
+python tools/quantize.py \
+		configs/detection/yolox/yolox_s_8xb16_300e_coco.py \
+		models/yolox_s.pth \
+		--device cpu \
+		--backend PyTorch
+```
```
Comment (wenmengzhou) [inline docs/source/tutorials/compression.md:59]: what is the path of the quantized_model?

File: docs/source/tutorials/compression.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+- `PRUNING_ALGORITHM`: pruning algorithm using by pruning class
+</details>
+
+**Examples:**
+
+Edit `data_root`path in the `${CONFIG_PATH}` to your own data path.
+
+```shell
+python tools/quantize.py \
```
Comment (wenmengzhou) [inline docs/source/tutorials/compression.md:92]: tools/prune.py

what is the path of the pruned_model?

File: docs/source/tutorials/compression.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
+# YOLOX Compression Tutorial
+
+## Data preparation
```
Comment (wenmengzhou) [inline docs/source/tutorials/compression.md:3]: add env preparation section to show necessary installtion steps

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**